code/modules/cm_aliens/structures/special/egg_morpher.dm

```diff
@@ -125,12 +125,6 @@
 			visible_message(SPAN_DANGER("\The [src] groans as its contents are reduced to nothing!"))
 			vis_contents.Cut()
 
-			for(var/atom/movable/A in captured_mob.contents_recursive()) // Get rid of any unacidable objects so we don't delete them
-				if(isobj(A))
-					var/obj/O = A
-					if(O.unacidable)
-						O.forceMove(get_turf(loc))
-
```

Comment:
...and preserve _all of this as is_, except replace the forcemove to the turf with that `O.forceMove(get_step(loc, pick(alldirs)))